source/js/nav-newsletter.js

```diff
@@ -163,7 +163,8 @@ class NavNewsletter {
       props.signupId || 0
     }/`;
     props.isHidden = false;
-    this.form = ReactDOM.render(<JoinUs {...props} />, elements.joinUs);
+    const root = createRoot(elements.joinUs);
+    this.form = root.render(<JoinUs {...props} />);
```

Comment:
I'm getting this error. See STR below

![image](https://user-images.githubusercontent.com/2896608/223251906-4f9523a7-c773-44c9-bc48-2149249a3b37.png)

### STR

- click on "Newsletter" button on nav
- click on that button again to collaspe the panel
- see error